tests/hwtimer/main.c

```diff
@@ -25,10 +25,12 @@
 #include "hwtimer.h"
 #include "thread.h"
```

Comment:
Now that you are at it: With removing `thread_sleep();` this include gets superfluous right?